<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="main_tab">
      <attribute name="title">
       <string>IDMS-Main</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="roi_tab">
      <attribute name="title">
       <string>ROI Generator</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="segment_tab">
      <attribute name="title">
       <string>Segmentation</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
